<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="employeeSection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Employee</string>
    </property>
    <widget class="QPushButton" name="viewDefectsButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Defects</string>
     </property>
    </widget>
    <widget class="QLabel" name="dashboardLabel_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dashboard</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="defectManagementSection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>Defects</string>
    </property>
    <widget class="QLineEdit" name="defectTitleInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="defectDescriptionInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>781</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="priorityComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="addDefectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewDefectsButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>520</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton_3">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>520</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loginSection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QLineEdit" name="usernameInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginErrorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addUserSection">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Add User</string>
    </property>
    <widget class="QComboBox" name="roleComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Employee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="newPasswordInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="newUsernameInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addUserConfirmButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToAdminButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="adminSection">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Admin</string>
    </property>
    <widget class="QPushButton" name="addUserButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manageDefectsButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Defects</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="dashboardLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dashboard</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
